noxfile.py

```diff
@@ -96,7 +101,7 @@ def test(session: nox.Session):
 
 @nox.session
 def coverage(session: nox.Session):
-    install(session, "coverage[toml]")
+    install(session, "coverage[toml]", *other_antsibull())
```

Comment:
`*other_antsibull()` shouldn't be needed just to run coverage.

Reply:
Ah sorry, I thought it also says `.[coverage]` here as in antsibull-changelog.

Reply:
52a8e59af928a77f1e12555f963b5029b2167215.